### DISCUSSION TITLE
Development Containers / GitHub Codespaces

### DISCUSSION THREAD

**Discussion by Alkarex** [FreshRSS/FreshRSS#4861] [Show and tell]
I have [just deployed](https://github.com/FreshRSS/FreshRSS/pull/4859) a test of Development Containers / GitHub Codespaces:

* https://containers.dev/
* https://github.com/features/codespaces

This allows having a full development environment, fully configured and ready for coding in one click, either in the Web browser or locally (e.g., in Visual Studio Code)

Give it a try ;-)

![image](https://user-images.githubusercontent.com/1008324/202221417-15bc2f89-5655-4e72-a4aa-ac30d3d93bba.png)

I have not yet finalised proper git signing (GPG) and other goodies, but potentially more to come, especially if there is interest.

@math-GH There you can run `make test-all` out of the box ;-)

![image](https://user-images.githubusercontent.com/1008324/202223352-93b71201-49c1-483b-b057-3f08cebf9b65.png)

Also with a FreshRSS instance running (see the *ports* tab):

![image](https://user-images.githubusercontent.com/1008324/202223668-b5a69a1f-2d68-4d99-b1ea-c7afcd534be6.png)

**Comment by math-GH** [comment]
Nice! github.dev has no terminal. It is "just" an editor.

How can I start the FreshRSS instance?

**Reply by Alkarex** [reply]
See screenshot higher up, there is a built-in terminal. See the *port* tab where there is a FreshRSS instance already running and exposed.
I can post more info / screenshots later if needed
